src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -787,7 +789,7 @@ Type type(Context context, boolean definition) throws ParserException {
             } else if (token.match(Token.ENUM, Token.EXPLICIT, Token.EXTERN, Token.INLINE, Token.CLASS, Token.FINAL,
                                    Token.INTERFACE, Token.__INTERFACE, Token.MUTABLE, Token.NAMESPACE, Token.STRUCT, Token.UNION,
                                    Token.TYPENAME, Token.REGISTER, Token.THREAD_LOCAL, Token.VOLATILE)) {
-                token = tokens.next();
+                tokens.next();
```

Comment:
That sounds like it could change the behavior. Let's keep those kinds of changes out.

Reply:
Or maybe not...

Reply:
The returned token is not used indeed.